src/Components/Server/test/Circuits/RemoteJSDataStreamTest.cs

```diff
@@ -154,10 +155,104 @@ public async void ReceiveData_ProvidedWithOutOfOrderChunk_SimulatesSignalRDiscon
             Assert.Equal("Out of sequence chunk received, expected 5, but received 7.", ex.Message);
         }
 
+        [Fact]
+        public async void ReceiveData_NoDataProvidedBeforeTimeout_StreamDisposed()
+        {
+            // Arrange
+            var jsRuntime = new TestRemoteJSRuntime(Options.Create(new CircuitOptions()), Options.Create(new HubOptions()), Mock.Of<ILogger<RemoteJSRuntime>>());
+            var timeoutExceptionRaisedSemaphore = new SemaphoreSlim(initialCount: 0, maxCount: 1);
+            jsRuntime.UnhandledException += (_, ex) =>
+            {
+                Assert.Equal("Did not receive any data in the alloted time.", ex.Message);
```

Comment:
```suggestion
                Assert.Equal("Did not receive any data in the allotted time.", ex.Message);
```

Presumably a typo in the underlying message.